src/app/api/crud/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .base import *
+from . import accesses
+from . import alerts
```

Comment:
Shouldn't it be:
```
from .accesses import *
from .alerts import *
```
keeping the same syntax?

Reply:
The 2 approaches have pros & cons: your suggestion implies we cannot have the same function name in 2 different subpackages because there would be a conflict here. Also I think it is clearer to use `crud.alerts.{something I know is related to alerts}`, but it is just my opinion ;)

Reply:
Fair enough, I just thought that since `base` had all the "shared" operations, each submodule would have functions name differently, but I'm fine keeping this as well!